### PR TITLE
Migrate from Jasmine gem to jasmine-browser-runner

### PR DESCRIPTION
**Pull request by baisa**
The Jasmine Ruby gems have been deprecated and are not compatible with Rails 7, which is now becoming a blocker.

[Trello card](https://trello.com/c/21VMx1NV/264-migrate-apps-to-use-jasmine-browser-runner)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "devDependencies": {
+    "jasmine-browser-runner": "^1.0.0",
+    "jasmine-core": "^4.0.1"
+  },
+  "scripts": {
+    "jasmine:prepare": "RAILS_ENV=test bundle exec rails assets:clobber assets:precompile",
+    "jasmine:ci": "yarn run jasmine:prepare && yarn run jasmine-browser-runner runSpecs",
+    "jasmine:browser": "yarn run jasmine:prepare && yarn run jasmine-browser-runner"
+  }
```
Comment (kevindew) [inline package.json:10]: We could use some more of the conventional package.json information in here. 

For example:

```
  "name": "transition",
  "description": "GOV.UK application to manage the transition of websites to GOV.UK",
  "private": true,
  "author": "Government Digital Service",
  "license": "MIT",
```

**Review by kevindew — COMMENTED** [review]
Nice job - there's just a few bits to fix but it looks functionally good.

File: package.json (hunk 1 above)
Comment (baisa) [reply package.json:15]: It is done 👍

**Review by kevindew — APPROVED** [review]
Nice one - looks good to me

**Pull request merged**